docs/en/guides/How-to-build.md

```diff
@@ -16,7 +16,7 @@ If you need to execute build behind the proxy, edit the *.mvn/jvm.config* and se
 ```
 
 ### Building from GitHub
-1. Prepare git, JDK11+, and Maven 3.6+.
+1. Prepare git, JDK 11/17/21(LTS versions), and Maven 3.6+.
```

Comment:
Adding 21 here makes no sense to me. Packages built with Java 21 is totally unusable, it's a waste of time for the users, and it adds up unnecessary potential issues that report this problem.

Reply:
21 has been removed.